app/components/EmbedStyling.tsx

```diff
@@ -57,6 +58,7 @@ body {
   background-color: transparent !important;
 }
 html, body.light-mode, body.dark-mode {${colorTheme}}
+.reverse-theme {${reverseTheme}}
```

Comment:
Two questions:
1. Is this necessary/correct here?
2. Will this page be interacting with the styles in `_app.tsx`? That is, do I need to override `body.light-mode .reverse-theme` etc., or does just specifying `.reverse-theme` suffice?

Sorry, I wasn't quite sure how to test this. Where can I access the pages where this is used? And how do they get composed?

Reply:
Update: tested locally and it seems like things work as is. I forgot about the (relatively recent) addition of the `key="theme"` property on both style tags (here and in _app.tsx) - when nextjs hoists these to the document head they use that key for uniqueness, so this one completely overrides the other ( https://nextjs.org/docs/pages/api-reference/components/head#avoid-duplicated-tags )

Reply:
Thanks for explaining this and looking into it 🙏